manifests/base/kustomization.yaml

```diff
@@ -15,3 +15,18 @@ resources:
 - ./redis
 - ./notification
 - ./applicationset-controller
+
```

Comment:
I had to create this at the top level.  With replacements the source path must resolve to a single source.